ansible/roles/prometheus/templates/prometheus.yml

```diff
@@ -248,4 +248,40 @@ scrape_configs:
         replacement: "/$2"
         target_label: "__metrics_path__"
         action: "replace"
+
+  # Scrape tasks in ECS using file based discovery, useful for application level metrics
+  - job_name: "ecs-tasks-dev"
```

Comment:
If possible, it would be better if we could use the same job name, but distinguish the `dev` and `prod` environments using a label instead of the job name, similar to how it's done for `ooniapi-services`, which will probably get replaced by this (see: https://github.com/ooni/devops/blob/e1ef1fcca0d11c600693188163d6bec7f108d7da/ansible/roles/prometheus/templates/prometheus.yml#L182)

Reply:
The problem here is the password part, we need two jobs because the metrics path in the hosts requires basic auth and the password is different in dev and prod, but we can only provide one password. But I can add relabel config to rename the job name or add an environment label

Reply:
In fact now I notice it's a bit troublesome to monitor both prod and dev right now because:
1. There's no clickhouse proxy in prod right now
2. The script  detects the environment in which it has permissions through the credentials that are created with Terraform, so we have to create a new cronjob with different credentials just for prod. Same thing with the configurations file and the wrapper file